src/index.html

```diff
@@ -31,7 +35,145 @@
     </nav>
   </header>
 
-  <main></main>
+  <main>
+    <section class="p-keyvisual__top-photo">
+      <ul>
+        <li>
+          <img src="../src/img/picture/keyvisual-message.png" alt="2021年使ってよかった転職サイトNO.1">
```

Comment:
index.html も src 配下にあるので、`src` は不要かと思いましたがいかがでしょう？
他の箇所も同様です。

```suggestion
          <img src="./img/picture/keyvisual-message.png" alt="2021年使ってよかった転職サイトNO.1">
```

Reply:
他の個所含めて修正しました。